src/services/token-interceptor.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, Injector } from '@angular/core';
+import { HttpInterceptor} from '@angular/common/http';
+import { UserService } from 'src/services/user.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TokenInterceptorService implements HttpInterceptor{
+
+  constructor(private injector:Injector) { }
+
+  intercept(req, next){
+    let userService=this.injector.get(UserService);
+    let tokenizedReq=req.clone({
+      setHeaders:{
+        Authorization: `Barber ${userService.getToken()}`
```

Comment:
add check if there is token... if there is none, value will be false, in case if there is no token - redirect to login page